src/animals/Antelope.js

```diff
@@ -0,0 +1,57 @@
+import { Animal } from '../organisms/Animal';
+import { getRandom } from '../utilities/getRandom';
+
+export class Antelope extends Animal {
+  constructor(positionY, positionX, age = 0) {
+    super(4, 4, positionY, positionX, age);
+  }
+
+  async action(board) {
+    // Double range of movement
+    const directions = [];
+    for (let directionY = -2; directionY <= 2; directionY++) {
+      for (let directionX = -2; directionX <= 2; directionX++) {
+        if (directionX === 0 && directionY === 0) continue;
+        directions.push({ positionY: directionY, positionX: directionX });
+      }
+    }
+
+    const availableDirections = directions.filter((direction) => {
+      const newY = this.positionY + direction.positionY;
+      const newX = this.positionX + direction.positionX;
+      return board.isValidPosition(newY, newX);
+    });
+
+    if (availableDirections.length > 0) {
+      const randomDirection =
+        availableDirections[getRandom(availableDirections.length)];
+      const newY = this.positionY + randomDirection.positionY;
+      const newX = this.positionX + randomDirection.positionX;
+
+      const targetOrganism = board.getOrganism(newY, newX);
+      if (!targetOrganism) {
+        board.moveOrganism(this, newY, newX);
+      } else if (targetOrganism.constructor.name === this.constructor.name) {
+        this.mate(board, targetOrganism);
+      } else {
+        // 50% chance to flee from fight
+        if (Math.random() < 0.5) {
```

Comment:
You use `Math.random() < x` multiple times throughout your code. It should be a separate function.

Reply:
```
function tryWithChance(fractionChance) {
  return Math.random() < fractionChance;
}
```